cmd/crates/soroban-test/tests/it/integration/tx.rs

```diff
@@ -0,0 +1,40 @@
+use soroban_cli::commands::tx;
+use soroban_sdk::xdr::{
+    Limits, ReadXdr, TransactionEnvelope, TransactionV1Envelope, VecM, WriteXdr,
+};
+use soroban_test::{AssertExt, TestEnv};
+
+use crate::integration::util::{deploy_contract, DeployKind, HELLO_WORLD};
+
+#[tokio::test]
+async fn txn_simulate() {
+    let sandbox = &TestEnv::new();
+    let xdr_base64_build_only = deploy_contract(sandbox, HELLO_WORLD, DeployKind::BuildOnly).await;
+    let xdr_base64_sim_only = deploy_contract(sandbox, HELLO_WORLD, DeployKind::SimOnly).await;
+    println!("{xdr_base64_build_only}");
```

Comment:
Should we remove this?

Reply:
It's only printed if it fails, but it was just for debugging.